x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -565,7 +565,6 @@
     "data.filter.filterBar.filterItemBadgeIconAriaLabel": "削除",
     "data.filter.filterBar.includeFilterButtonLabel": "結果を含める",
     "data.filter.filterBar.indexPatternSelectPlaceholder": "インデックスパターンの選択",
-    "data.filter.filterBar.labelErrorMessage": "フィルターを表示できませんでした",
```

Comment:
Why is this removed? 🤔 I see `data.filter.filterBar.labelErrorMessage` still used in `src/plugins/data/public/ui/filter_bar/filter_view/index.tsx`

Reply:
As far as I see, it's not being used anymore

Reply:
My bad.. mistaken with `'data.filter.filterBar.moreFilterActionsMessage'`